Makefile

```diff
@@ -177,10 +177,16 @@ cadenceIntegTests:
 	$Q go test -v ./integ -temporal=false
 
 ci-cadence-integ-test:
-	$Q go test -v ./integ -search=false -temporal=false -dependencyWaitSeconds=180
+	$Q go test -v ./integ -search=false -temporal=false -dependencyWaitSeconds=180 -disableStickyCache=true
+
+ci-cadence-integ-test-disable-sticky:
+	$Q go test -v ./integ -run "(?i)^Test[${startsWith}]" -search=false -temporal=false -dependencyWaitSeconds=180 -disableStickyCache=true
 
 ci-temporal-integ-test:
```

Comment:
Can you test does `-disableStickyCache=true` work? I thought it has to be `-disableStickyCache` because it's a bool flag

Reply:
It does work, but I can simplify it to `disableStickyCache`

Reply:
`-disableStickyCache` by itself does not work, so I'm reverting to `-disableStickyCache=true`

Reply:
Done